ecosystem/sep-0001.md

```diff
@@ -69,6 +69,18 @@ DEPOSIT_SERVER="https://api.domain.com"
 # The signing key is used for the compliance protocol
 SIGNING_KEY="GBBHQ7H4V6RRORKYLHTCAWP6MOHNORRFJSDPXDFYDGJB2LPZUFPXUEW3"
 
+# Name of your organization for display purposes
+ORG_NAME = "Organization Name"
+
+# URL where people can learn more about your organization
+ORG_URL = "http://www.domain.com"
+
+# Your organization's logo
+ORG_LOGO = "http://www.domain.com/awesomelogo.jpg"
+
+# Short description of your organization
+ORG_DESCRIPTION = "Description of issuer"
```

Comment:
Looks good but I'd rename it to `ORGANIZATION_NAME`, `ORGANIZATION_URL` and. We don't use abbreviations anywhere else in stellar.toml file.